main/ZgatewayBT.ino

```diff
@@ -263,7 +263,7 @@ void CLEARGRASSTRHDiscovery(char* mac) {
       //component type,name,availability topic,device class,value template,payload on, payload off, unit of measurement
   };
 
-  createDiscoveryFromList(mac, CLEARGRASSTRHsensor, CLEARGRASSTRHparametersCount);
+  createDiscoveryFromList(mac, CLEARGRASSTRHsensor, CLEARGRASSTRHparametersCount, "", "ClearGrass", "");
```

Comment:
You can find all the model reference below:
https://github.com/1technophile/OpenMQTTGateway/blob/e17b5b9c9d97ac83ef51aefdf8c93de408d8046e/main/ZgatewayBT.ino#L836

Reply:
fixed